index.js

```diff
@@ -111,18 +114,144 @@ router.get("/", async (req, res, next) => {
 
 });
 
+router.get("/authorization-code/default",
```

Comment:
This should be handled by the tenantResolver registerTenantRoutes

Reply:
Removed